underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -405,6 +405,9 @@ private String stripFolderSuffixIfPresent(String key) {
    * @return the key without the Swift container prefix
    */
   private String stripPrefixIfPresent(String path) {
+    if (path.startsWith(PATH_SEPARATOR)) {
+      return stripPrefixIfPresent(path, PATH_SEPARATOR);
+    }
```

Comment:
I think because of JOSS, it always adds a /, so if don't trim the first /, then there will be double / there.
The Alluxio code tries to trim only "swift://bucket/", there will be an invalid path error if the user tries to input something like "swift://bucket", which might be fixed from the Alluxio code side.